Resources/public/js/views/services/ez-dashboardblockbasedviewservice.js

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-dashboardblockbasedviewservice', function (Y) {
+    'use strict';
+
+    /**
+     * Provides the dashboard block based view service class
+     *
+     * @method ez-dashboardblockbasedviewservice
+     */
+    Y.namespace('eZ');
+
+    var DEFAULT_ERROR_IDENTIFIER = 'dashboardblockbasedviewservice-error',
+        DEFAULT_ERROR_MESSAGE = 'DashboardBlockBasedViewService: error';
+
+    /**
+     * The dashboard block based view service
+     *
+     * @namespace eZ
+     * @class DashboardBlockBasedViewService
+     * @constructor
+     * @extends eZ.ViewService
+     */
+    Y.eZ.DashboardBlockBasedViewService = Y.Base.create('dashboardBlockBasedViewService', Y.eZ.ViewService, [], {
+        _load: function (next) {
+            this._loadRootLocation()
```

Comment:
At this step, it's too early to decide that. At the moment, those Locations are used any where => useless code => remove code.